WARNING 2023-10-03 03:32:00,441 log Not Found: /robots.txt
WARNING 2023-10-03 03:33:03,576 log Not Found: /Core/Skin/Login.aspx
WARNING 2023-10-03 05:06:39,419 log Not Found: /Core/Skin/Login.aspx
WARNING 2023-10-03 05:12:51,587 log Not Found: /robots.txt
WARNING 2023-10-03 05:28:45,136 log Not Found: /robots.txt
WARNING 2023-10-03 05:44:08,497 log Not Found: /shop/nl/moncler-women-down-coats-designer-metallic-fabric-black/
WARNING 2023-10-03 05:52:46,968 log Not Found: /robots.txt
WARNING 2023-10-03 06:04:47,117 log Not Found: /p.php
WARNING 2023-10-03 06:04:47,136 log Not Found: /t.php
WARNING 2023-10-03 06:04:47,160 log Not Found: /by.php
ERROR 2023-10-03 06:29:23,074 log Internal Server Error: /inquiry_details/
Traceback (most recent call last):
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/github/easygo/basecamp/views.py", line 481, in inquiry_details
    p.save()
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/db/models/base.py", line 806, in save
    self.save_base(
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/db/models/base.py", line 857, in save_base
    updated = self._save_table(
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/db/models/base.py", line 1000, in _save_table
    results = self._do_insert(
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/db/models/base.py", line 1041, in _do_insert
    return manager._insert(
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/db/models/query.py", line 1434, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/db/models/sql/compiler.py", line 1620, in execute_sql
    for sql, params in self.as_sql():
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/db/models/sql/compiler.py", line 1547, in as_sql
    value_rows = [
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/db/models/sql/compiler.py", line 1548, in <listcomp>
    [
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/db/models/sql/compiler.py", line 1549, in <listcomp>
    self.prepare_value(field, self.pre_save_val(field, obj))
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/db/models/sql/compiler.py", line 1487, in prepare_value
    value = field.get_db_prep_save(value, connection=self.connection)
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/db/models/fields/__init__.py", line 910, in get_db_prep_save
    return self.get_db_prep_value(value, connection=connection, prepared=False)
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/db/models/fields/__init__.py", line 1408, in get_db_prep_value
    value = self.get_prep_value(value)
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/db/models/fields/__init__.py", line 1403, in get_prep_value
    return self.to_python(value)
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/db/models/fields/__init__.py", line 1369, in to_python
    raise exceptions.ValidationError(
django.core.exceptions.ValidationError: ['“R C Jp” value has an invalid date format. It must be in YYYY-MM-DD format.']
WARNING 2023-10-03 06:33:26,698 log Not Found: /shop/de/product/moncler-jacken-manner-zin-schwarz/
WARNING 2023-10-03 06:39:58,247 log Not Found: /Core/Skin/Login.aspx
WARNING 2023-10-03 06:55:56,163 log Not Found: /shop/product/moncler-jackets-men-quentin-khaki/
WARNING 2023-10-03 07:02:48,542 log Not Found: /wp-content/plugins/WordPressSeo/include.php
WARNING 2023-10-03 07:04:33,294 log Not Found: /wp-content/plugins/WordPressCore/include.php
WARNING 2023-10-03 07:21:20,035 log Not Found: /contact/
WARNING 2023-10-03 07:23:52,382 log Not Found: /owa/auth/logon.aspx
WARNING 2023-10-03 07:47:22,711 log Not Found: /ab2g
WARNING 2023-10-03 07:47:23,243 log Not Found: /ab2h
WARNING 2023-10-03 07:47:27,496 log Not Found: /t4
WARNING 2023-10-03 08:13:15,233 log Not Found: /Core/Skin/Login.aspx
WARNING 2023-10-03 08:13:44,852 log Not Found: /.env
WARNING 2023-10-03 08:13:46,237 log Forbidden (Referer checking failed - no Referer.): /
WARNING 2023-10-03 08:25:50,588 log Not Found: /robots.txt
WARNING 2023-10-03 08:25:53,618 log Not Found: /app.env.json
WARNING 2023-10-03 08:37:03,448 log Not Found: /prod-secrets.env
WARNING 2023-10-03 08:53:35,344 log Not Found: /robots.txt
WARNING 2023-10-03 09:29:40,165 log Not Found: /robots.txt
WARNING 2023-10-03 09:31:42,658 log Not Found: /2022/wp-includes/css/admin-bar-rtl.css
WARNING 2023-10-03 09:32:58,784 log Not Found: /wp-includes/css/admin-bar-rtl.css
WARNING 2023-10-03 09:33:00,636 log Not Found: /cms/wp-includes/css/admin-bar-rtl.css
WARNING 2023-10-03 09:33:04,835 log Not Found: /robots.txt
WARNING 2023-10-03 09:33:06,372 log Not Found: /humans.txt
WARNING 2023-10-03 09:33:07,962 log Not Found: /ads.txt
WARNING 2023-10-03 09:34:40,366 log Not Found: /shop/wp-includes/css/admin-bar-rtl.css
WARNING 2023-10-03 09:34:56,671 log Not Found: /wp1/wp-includes/css/admin-bar-rtl.css
WARNING 2023-10-03 09:35:07,562 log Not Found: /site/wp-includes/css/admin-bar-rtl.css
WARNING 2023-10-03 09:35:11,997 log Not Found: /news/wp-includes/css/admin-bar-rtl.css
WARNING 2023-10-03 09:35:35,958 log Not Found: /wordpress/wp-includes/css/admin-bar-rtl.css
WARNING 2023-10-03 09:36:31,014 log Not Found: /wp2/wp-includes/css/admin-bar-rtl.css
WARNING 2023-10-03 09:37:38,780 log Not Found: /wp/wp-includes/css/admin-bar-rtl.css
WARNING 2023-10-03 09:39:34,467 log Not Found: /test/wp-includes/css/admin-bar-rtl.css
WARNING 2023-10-03 09:40:05,670 log Not Found: /2021/wp-includes/css/admin-bar-rtl.css
WARNING 2023-10-03 09:40:22,809 log Not Found: /sito/wp-includes/css/admin-bar-rtl.css
WARNING 2023-10-03 09:40:27,439 log Not Found: /media/wp-includes/css/admin-bar-rtl.css
WARNING 2023-10-03 09:42:01,218 log Not Found: /2018/wp-includes/css/admin-bar-rtl.css
WARNING 2023-10-03 09:42:16,150 log Not Found: /web/wp-includes/css/admin-bar-rtl.css
WARNING 2023-10-03 09:43:14,050 log Not Found: /2020/wp-includes/css/admin-bar-rtl.css
WARNING 2023-10-03 09:43:47,704 log Not Found: /blog/wp-includes/css/admin-bar-rtl.css
WARNING 2023-10-03 09:44:06,826 log Not Found: /website/wp-includes/css/admin-bar-rtl.css
WARNING 2023-10-03 09:46:25,567 log Not Found: /Core/Skin/Login.aspx
WARNING 2023-10-03 09:47:47,192 log Not Found: /2019/wp-includes/css/admin-bar-rtl.css
WARNING 2023-10-03 10:27:23,629 log Not Found: /shop/it/product/moncler-giacche-bambini-daim-rose/
WARNING 2023-10-03 10:40:18,995 log Not Found: /robots.txt
WARNING 2023-10-03 10:45:43,672 log Not Found: /app_secrets.json
WARNING 2023-10-03 11:01:15,956 log Not Found: /robots.txt
WARNING 2023-10-03 11:08:49,777 log Not Found: /wp-content/plugins/WordPressCore/include.php
WARNING 2023-10-03 11:08:56,826 log Not Found: /wp-content/plugins/core-plugin/include.php
WARNING 2023-10-03 11:09:09,259 log Not Found: /wp-includes/widgets/include.php
WARNING 2023-10-03 11:09:23,171 log Not Found: /wp-includes/images/include.php
WARNING 2023-10-03 11:09:34,284 log Not Found: /wp-content/admin.php
WARNING 2023-10-03 11:09:47,235 log Not Found: /wp-content/plugins/about.php
WARNING 2023-10-03 11:09:56,866 log Not Found: /geju.php
WARNING 2023-10-03 11:10:09,890 log Not Found: /ayk.php
WARNING 2023-10-03 11:10:20,232 log Not Found: /wp.php
WARNING 2023-10-03 11:19:34,168 log Not Found: /Core/Skin/Login.aspx
WARNING 2023-10-03 11:23:42,160 log Not Found: /wp-content/plugins/WordPressCore/include.php
WARNING 2023-10-03 11:23:57,531 log Not Found: /wp-content/plugins/core-plugin/include.php
WARNING 2023-10-03 11:24:09,267 log Not Found: /wp-includes/widgets/include.php
WARNING 2023-10-03 11:24:23,784 log Not Found: /wp-includes/images/include.php
WARNING 2023-10-03 11:24:35,569 log Not Found: /wp-content/admin.php
WARNING 2023-10-03 11:24:58,587 log Not Found: /wp-content/plugins/about.php
WARNING 2023-10-03 11:25:07,169 log Not Found: /geju.php
WARNING 2023-10-03 11:25:23,838 log Not Found: /ayk.php
WARNING 2023-10-03 11:25:55,045 log Not Found: /wp.php
WARNING 2023-10-03 11:27:22,847 log Not Found: /robots.txt
WARNING 2023-10-03 11:28:18,701 log Not Found: /chatswood/
WARNING 2023-10-03 11:30:54,950 log Not Found: /robots.txt
WARNING 2023-10-03 11:47:06,545 log Not Found: /robots.txt
WARNING 2023-10-03 11:47:06,978 log Not Found: /ads.txt
WARNING 2023-10-03 12:00:36,197 log Not Found: /robots.txt
ERROR 2023-10-03 12:10:52,588 log Internal Server Error: /reminder_detail/
Traceback (most recent call last):
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/github/easygo/basecamp/views.py", line 1785, in reminder_detail
    sending_reminder_email()
TypeError: sending_reminder_email() missing 1 required positional argument: 'user'
ERROR 2023-10-03 12:16:54,847 log Internal Server Error: /reminder_detail/
Traceback (most recent call last):
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/github/easygo/basecamp/views.py", line 1785, in reminder_detail
    sending_reminder_email(user1.email)
  File "/home/ubuntu/github/easygo/basecamp/views.py", line 1757, in sending_reminder_email
    {'name': user.name, 'email': user.email})
AttributeError: 'str' object has no attribute 'name'
ERROR 2023-10-03 12:18:48,741 log Internal Server Error: /reminder_detail/
Traceback (most recent call last):
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/github/easygo/basecamp/views.py", line 1785, in reminder_detail
    sending_reminder_email(user.email)
  File "/home/ubuntu/github/easygo/basecamp/views.py", line 1757, in sending_reminder_email
    {'name': user.name, 'email': user.email})
AttributeError: 'str' object has no attribute 'name'
ERROR 2023-10-03 12:19:43,027 log Internal Server Error: /reminder_detail/
Traceback (most recent call last):
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/github/easygo/basecamp/views.py", line 1785, in reminder_detail
    sending_reminder_email(user.email)
  File "/home/ubuntu/github/easygo/basecamp/views.py", line 1757, in sending_reminder_email
    {'name': user.name, 'email': user.email})
AttributeError: 'str' object has no attribute 'name'
ERROR 2023-10-03 12:22:49,570 log Internal Server Error: /reminder_detail/
Traceback (most recent call last):
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/github/easygo/basecamp/views.py", line 1786, in reminder_detail
    sending_reminder_email(user.email)
  File "/home/ubuntu/github/easygo/basecamp/views.py", line 1756, in sending_reminder_email
    user = Post.objects.get(email=user.email)
AttributeError: 'str' object has no attribute 'email'
ERROR 2023-10-03 12:24:02,111 log Internal Server Error: /reminder_detail/
Traceback (most recent call last):
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/github/easygo/basecamp/views.py", line 1785, in reminder_detail
    sending_reminder_email(user.email)
  File "/home/ubuntu/github/easygo/basecamp/views.py", line 1757, in sending_reminder_email
    {'name': user.name, 'email': user.email})
AttributeError: 'str' object has no attribute 'name'
ERROR 2023-10-03 12:24:46,577 log Internal Server Error: /reminder_detail/
Traceback (most recent call last):
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/github/easygo/basecamp/views.py", line 1785, in reminder_detail
    sending_reminder_email()
TypeError: sending_reminder_email() missing 1 required positional argument: 'user'
ERROR 2023-10-03 12:29:53,077 log Internal Server Error: /reminder_detail/
Traceback (most recent call last):
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/github/easygo/basecamp/views.py", line 1775, in reminder_detail
    user1 = Post.objects.filter(email=email)[1]
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/db/models/query.py", line 362, in __getitem__
    return qs._result_cache[0]
IndexError: list index out of range
WARNING 2023-10-03 12:50:20,082 log Not Found: /airport-transfers-leichardt/
WARNING 2023-10-03 12:52:27,128 log Not Found: /Core/Skin/Login.aspx
WARNING 2023-10-03 12:57:44,743 log Not Found: /home
ERROR 2023-10-03 13:01:20,799 log Internal Server Error: /reminder_detail/
Traceback (most recent call last):
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/github/easygo/basecamp/views.py", line 1776, in reminder_detail
    if user.return_pickup_time =='x':
AttributeError: 'QuerySet' object has no attribute 'return_pickup_time'
ERROR 2023-10-03 13:02:16,056 log Internal Server Error: /reminder_detail/
Traceback (most recent call last):
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/github/easygo/basecamp/views.py", line 1776, in reminder_detail
    if user.return_pickup_time =='x':
AttributeError: 'QuerySet' object has no attribute 'return_pickup_time'
ERROR 2023-10-03 13:03:06,595 log Internal Server Error: /reminder_detail/
Traceback (most recent call last):
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
  File "/home/ubuntu/github/easygo/venv/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/github/easygo/basecamp/views.py", line 1776, in reminder_detail
    if user.return_pickup_time =='x':
AttributeError: 'QuerySet' object has no attribute 'return_pickup_time'
WARNING 2023-10-03 13:45:40,437 log Not Found: /robots.txt
WARNING 2023-10-03 13:47:13,583 log Not Found: /php.ini
WARNING 2023-10-03 13:56:01,292 log Not Found: //.env
WARNING 2023-10-03 14:03:09,615 log Not Found: /owa/auth/x.js
WARNING 2023-10-03 14:04:28,720 log Not Found: /shop/product/moncler-jackets-men-leblond-khaki/
WARNING 2023-10-03 14:11:53,059 log Forbidden (Referer checking failed - no Referer.): /
